src/rebloom.c

```diff
@@ -676,6 +678,11 @@ static int CFInsert_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
     if (items_pos < 0 || items_pos == argc) {
         return RedisModule_WrongArity(ctx);
     }
+
+    if (options.capacity < 1) {
```

Comment:
Please change to `2 * CF_DEFAULT_BUCKETSIZE`, similar to the limit in `CFReserve_RedisCommand` and move into the switch at the end of the case `c`.

Reply:
Done @ashtul  👀